app/models/introduction.rb

```diff
@@ -0,0 +1,2 @@
+class Introduction < ActiveRecord::Base
```

Comment:
You probably don't need a model for introduction - this can just be a text field in your user model ;)
you can delete models with `rails destroy model introduction`